examples/jsm/loaders/KTX2Loader.js

```diff
@@ -242,33 +242,33 @@ class KTX2Loader extends Loader {
 	load( url, onLoad, onProgress, onError ) {
 
 		if ( this.workerConfig === null ) {
-
-			throw new Error( 'THREE.KTX2Loader: Missing initialization with `.detectSupport( renderer )`.' );
-
+		  throw new Error( 'THREE.KTX2Loader: Missing initialization with `.detectSupport( renderer )`.' );
 		}
```

Comment:
The whitespace is intentional – unless your code editor makes this difficult, please do try to keep it. :)